src/components/visualizations/utils/drilldownEventing.ts

```diff
@@ -65,15 +66,20 @@ function composeDrillContextGroup(
     points: IHighchartsPointObject[],
     chartType: ChartType,
 ): IDrillEventContextGroup {
+    const isCombo = isComboChart(chartType);
     const contextPoints: IDrillPoint[] = points.map((point: IHighchartsPointObject) => {
+        const customProps: Partial<IDrillPoint> = isCombo ? { type: get(point, "series.type") } : {};
+
         return {
             x: point.x,
             y: point.y,
             intersection: point.drillIntersection,
+            ...customProps,
         };
     });
+
     return {
-        type: chartType,
+        type: getVisualizationType(chartType),
```

Comment:
Should correct chartType in `chartClickDebounced` function, then it can be reused on both `composeDrillContextPoint` and `composeDrillContextGroup` function